Algebra.java

```diff
@@ -6,62 +6,182 @@
+	    //System.out.println(pow(1,5));        // 1
+   		//System.out.println(div(12,3));   // 4    
+   		//System.out.println(div(-36,-6));    // 1  
+   		//System.out.println(div(25,7));   // 3
+	    //System.out.println(div(-6,2));      //-3
+		//System.out.println(div(6,-2));      //-3
+		//System.out.println(div(0,7));    // 0
+		//System.out.println(div(3,1));    // 3
+   		//System.out.println(mod(6,4));       // 25 % 7
+		//System.out.println(mod(6,3));       // 25 % 7
+		//System.out.println(mod(34,8));       // 2
+   		//System.out.println(mod(120,6));      // 120 % 6   
+		//System.out.println(mod(-8,6));
+		//System.out.println(mod(6,-2)); 
+   		//System.out.println(sqrt(0));
+		//System.out.println(sqrt(263169));
    		System.out.println(sqrt(76123));
+		System.out.println(sqrt(9));
+		System.out.println(sqrt(81));
 	}  
 
 	// Returns x1 + x2
 	public static int plus(int x1, int x2) {
-		// Replace the following statement with your code
-		return 0;
+		if(x2 < 0) {
+			for(int i = 0; i < Math.abs(x2); i++) {
+				x1--;
+			}
+		} else {
+			for(int i = 0; i < x2; i++) {
+				x1++;
+			}
+		}
+		return x1;
 	}
 
 	// Returns x1 - x2
 	public static int minus(int x1, int x2) {
-		// Replace the following statement with your code
-		return 0;
+		if(x2 < 0) {
+			for(int i = 0; i < Math.abs(x2); i++) {
+				x1++;
+			}
+		} else {
+			for(int i = 0; i < x2; i++) {
+				x1--;
+			}
+		}
+		return x1;
 	}
 
 	// Returns x1 * x2
 	public static int times(int x1, int x2) {
-		// Replace the following statement with your code
+		int mult = x1;
+		if(x1 == 1)
+		return x2;
+		else if(x2 == 1)
+		return x1;
+		else if(x1 == 0 || x2 == 0)
 		return 0;
+		else if(x1 == -1 && x2 == -1)
+		return 1;
+		else if(x1 == -1 && x2 == 1)
+		return -1;
+		else if(x1 == 1 && x2 == -1)
+		return -1;
+		else if(x1 == 1 && x2 == 1)
+		return 1;
+		else if(x1 < 0 && x2 < 0) {
+			for(int i = 1; i < Math.abs(x2); i++) {
+				mult = plus(mult, x1);
+			}
+			mult = Math.abs(mult);
+		} else if(x1 < 0 && x2 > 0) {
+			for(int i = 1; i < Math.abs(x2); i++) {
+				mult = plus(mult, x1);
+			}
+		} else if(x1 > 0 && x2 < 0) {
+			for(int i = 1; i < Math.abs(x2); i++) {
+				mult = plus(mult, x1);
+			}
+			mult = minus(0, mult);
+		} else {
+			for(int i = 1; i < Math.abs(x2); i++) {
+				mult = plus(mult, x1);
+			}
+		}
+		return mult;
 	}
 
 	// Returns x^n (for n >= 0)
 	public static int pow(int x, int n) {
-		// Replace the following statement with your code
-		return 0;
+		if(n == 0)
+		return 1;
+		if(n == 1)
+		return x;
+		int pow = x;
+		for(int i = 1; i < n; i++) {
+			pow = times(pow, x);
+		}
+		return pow;
 	}
 
 	// Returns the integer part of x1 / x2 
-	public static int div(int x1, int x2) {
-		// Replace the following statement with your code
+	public static int div(int x1, int x2) { 
+		if(x1 == 0)
 		return 0;
+		if(x2 == 1)
+		return x1;
+		int q = x1;
+		int count = 0;
+		if(x1 < 0 && x2 < 0){
+			x2 = Math.abs(x2);
+			x1 = Math.abs(x1);
```

Comment:
Math.abs(x2) is Math operation.. You are asked to work without Math...
This is violation of Algebra guidelines -2.